frontend/components/StateSelect/UsStateSelect.tsx

```diff
@@ -0,0 +1,59 @@
+import styles from './select.module.css'
+
+export function USStateSelect() {
+    return (
+      <select id="state" className={styles.input}>
```

Comment:
out of curiosity why not use a library for this?

Reply:
For the forms, or the state selection component specifically? At a glance, I'm unsure if pulling in a library for one component justifies the extra dependency we can avoid by just having this pasted in.

Reply:
Just un-resolving in case you missed my comment!

Reply:
Maybe? 

> I think this ticket has scope-crept pretty significantly - my understanding was this was a frontend ticket to create the UI of the Passport application page - but it seems only 4-7 of the 50 file changes have to do with that task
> 
> I think it'd probably be a good idea to break out the extra work you did into their own tickets/branches so that it's clear what we're trying to accomplish with each & so that the appropriate people can review your PR successfully
> 
> Screenshots/explanatory comments could also be helpful if it's not too much trouble!

Something is very wrong, there are a bunch of changes I didn't make, and I'm not sure why they're there. This will hopefully be more normalized once I pull in the latest changes from master.